File: problem10.py
#!/bin/python

import math

def is_prime(num):
	for i in xrange(2, int(math.sqrt(num))+1):
		if not num % i:
			return False
	return True
	
def get_sum_of_primes(limit):
	primes = []
	for num in range(2, limit):
		if is_prime(num):
			primes.append(num)

	return sum(primes)
	
if __name__ == '__main__':
	print "Answer found: ", get_sum_of_primes(2000000)

File: problem10.java
public class problem10 {

    public static boolean isPrime(int num) {

        int limit = new Double(Math.sqrt(num) + 1).intValue();

        for (int i = 2; i < limit; i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static void main(String args[]) {

        long sum = 1,
            max = 2000000;

        for (int i = 1; i < max; i += 2) {
            if (isPrime(i)) {
                sum += i;
            }
        }
        System.out.println("Answer found: " + sum);
    }
}
